From c35adbbf2ae6e8668951e2b5fbb9b2c7c0b158c1 Mon Sep 17 00:00:00 2001
From: Stefano Babic <sbabic@denx.de>
Date: Sun, 8 Jan 2017 17:39:16 +0100
Subject: [PATCH v1 1/4] Rename aes.h to uboot_aes.h

aes.h is a too generic name if this file can
be exported and used by a program.
Rename it to avoid any conflicts with
other files (for example, from openSSL).

Signed-off-by: Stefano Babic <sbabic@denx.de>

---

 arch/arm/mach-tegra/tegra20/crypto.c | 2 +-
 cmd/aes.c                            | 2 +-
 common/env_common.c                  | 2 +-
 include/{aes.h => uboot_aes.h}       | 0
 lib/aes.c                            | 2 +-
 tools/env/fw_env.h                   | 2 +-
 6 files changed, 5 insertions(+), 5 deletions(-)
 rename include/{aes.h => uboot_aes.h} (100%)

diff --git a/arch/arm/mach-tegra/tegra20/crypto.c b/arch/arm/mach-tegra/tegra20/crypto.c
index 1b82fbb..eae7921 100644
--- a/arch/arm/mach-tegra/tegra20/crypto.c
+++ b/arch/arm/mach-tegra/tegra20/crypto.c
@@ -8,7 +8,7 @@
 #include <common.h>
 #include <linux/errno.h>
 #include "crypto.h"
-#include "aes.h"
+#include "uboot_aes.h"
 
 static u8 zero_key[16];
 
diff --git a/cmd/aes.c b/cmd/aes.c
index 76da3ef..ee1ae13 100644
--- a/cmd/aes.c
+++ b/cmd/aes.c
@@ -9,7 +9,7 @@
 #include <common.h>
 #include <command.h>
 #include <environment.h>
-#include <aes.h>
+#include <uboot_aes.h>
 #include <malloc.h>
 #include <asm/byteorder.h>
 #include <linux/compiler.h>
diff --git a/common/env_common.c b/common/env_common.c
index 7fb62e8..6845f8d 100644
--- a/common/env_common.c
+++ b/common/env_common.c
@@ -140,7 +140,7 @@ int set_default_vars(int nvars, char * const vars[])
 }
 
 #ifdef CONFIG_ENV_AES
-#include <aes.h>
+#include <uboot_aes.h>
 /**
  * env_aes_cbc_get_key() - Get AES-128-CBC key for the environment
  *
diff --git a/include/aes.h b/include/uboot_aes.h
similarity index 100%
rename from include/aes.h
rename to include/uboot_aes.h
diff --git a/lib/aes.c b/lib/aes.c
index 9d7a0a1..d6144e6 100644
--- a/lib/aes.c
+++ b/lib/aes.c
@@ -27,7 +27,7 @@
 #else
 #include <string.h>
 #endif
-#include "aes.h"
+#include "uboot_aes.h"
 
 /* forward s-box */
 static const u8 sbox[256] = {
diff --git a/tools/env/fw_env.h b/tools/env/fw_env.h
index 05588ab..0d7130a 100644
--- a/tools/env/fw_env.h
+++ b/tools/env/fw_env.h
@@ -5,7 +5,7 @@
  * SPDX-License-Identifier:	GPL-2.0+
  */
 
-#include <aes.h>
+#include <uboot_aes.h>
 #include <stdint.h>
 
 /* Pull in the current config to define the default environment */
-- 
2.7.4

